projects/1.16.1/assets/tetra/tetra/lang/zh_cn.json

```
@@ -0,0 +1,2083 @@
+    "block.tetra.forged_wall": "锻造墙",
+    "block.tetra.forged_vent": "锻造通风口",
+    "block.tetra.forged_crate": "锻造板条箱",
+    "block.tetra.forged_container": "锻造容器",
+    "tetra.forged_container.compartment": "隔间 %s",
+    "block.tetra.hammer_base": "锻造锤",
+    "block.tetra.hammer_head.name": "锤头",
+    "block.tetra.forged_workbench": "锻造工作台",
+    "block.tetra.transfer_unit": "传输单元",
+    "block.tetra.core_extractor": "地核提取器",
+    "block.tetra.extractor_piston": "提取活塞",
+    "block.tetra.extractor_pipe": "提取管道",
+    "block.tetra.seeping_bedrock": "渗流基岩",
+    "block.multiblock_hint.1x2x1": "§8[§71x2x1 多方块§8]",
+    "item.tetra.forged_bolt": "螺栓",
+    "tetra.salvage_bolt_action.label": "回收",
+    "item.tetra.forged_beam": "结构横梁",
+    "tetra.salvage_beam_action.label": "回收",
+    "item.tetra.forged_mesh": "柔性网格",
+    "tetra.salvage_mesh_action.label": "回收",
+    "item.tetra.quick_latch": "快卸销",
+    "tetra.salvage_quick_latch_action_hammer.label": "回收",
+    "tetra.salvage_quick_latch_action_pry.label": "回收",
+    "item.tetra.metal_scrap": "金属废料",
+    "item.tetra.metal_scrap.description": "奇形怪状的金属，它们的并无用处或用途未知，等待你赋予它们全新的价值",
+    "tetra.salvage_metal_scrap_action.label": "回收",
+    "item.tetra.vent_plate": "隔热板",
+    "item.tetra.vent_plate.description": "连接到锻造锤或传输单元上能稍微提高效率",
+    "tetra.salvage_vent_plate_action.label": "回收",
+    "item.tetra.combustion_chamber": "燃烧室",
+    "item.tetra.combustion_chamber.description": "可以被连接到锻造锤上提升 1 级锤子的等级",
+    "tetra.salvage_combustion_chamber_action.label": "回收",
+    "item.tetra.planar_stabilizer": "平面稳定器",
+    "item.tetra.planar_stabilizer.description": "可以被连接到锻造锤上，让用锻造锤合成的主要组件略微增加属性",
+    "tetra.salvage_planar_stabilizer_action.label": "回收",
+    "item.tetra.stonecutter": "切石器",
+    "tetra.salvage_stonecutter_action.label": "回收",
+    "item.tetra.earthpiercer": "穿地器",
+    "tetra.salvage_earthpiercer_action.label": "回收",
+    "item.tetra.forged_unsettling": "这件古物发出令人反胃的振动，它看起来被做成了装在手柄上的样子",
+    "item.tetra.forged_description": "能在下雪群系的地下深处找到",
+    "item.tetra.tooltip_reveal": "在物品栏中显示相关的升级",
+    "item.tetra.tooltip_expand": "§7[§8shift§7] §8+",
+    "item.tetra.tooltip_expanded": "§8[§fshift§8] §f+",
+    "tetra.slot.sword/blade": "剑刃",
+    "tetra.slot.sword/hilt": "剑柄",
+    "tetra.slot.sword/guard": "护手",
+    "tetra.slot.sword/pommel": "剑柄圆头",
+    "tetra.slot.sword/fuller": "附层",
+    "tetra.slot.double/head_left": "头部§7（左）",
+    "tetra.slot.double/head_right": "头部§7（右）",
+    "tetra.slot.double/handle": "手柄",
+    "tetra.slot.double/binding": "绑定",
+    "tetra.slot.double/accessory": "附件",
+    "item.tetra.modular.settle_progress.label": "固定进度",
+    "item.tetra.modular.settle_progress_gain.description": "当物品在被使用时这个组件会被缓慢地固定下来，当被固定后它会提供额外的完整度。\n \n§7进度：§e%s§7/§e%s\n \n§7基础要求：§e%s\n§7耐久惩罚：§e%s",
+    "item.tetra.modular.settle_progress_cost.description": "当物品在被使用时这个组件会被缓慢地固定下来，当被固定后它的完整度消耗会下降。\n \n§7进度：§e%s§7/§e%s\n \n§7基础要求：§e%s\n§7耐久惩罚：§e%s",
```

|Feedback Priority|
|---|
|⚪️ Pebble / Non-blocking but requires future action|

```suggestion
    "item.tetra.modular.settle_progress_gain.description": "物品被使用时这个组件会缓慢地固定下来，固定住后它会提供额外的完整度。\n \n§7进度：§e%s§7/§e%s\n \n§7基础要求：§e%s\n§7耐久惩罚：§e%s",
    "item.tetra.modular.settle_progress_cost.description": "物品被使用时这个组件会缓慢地固定下来，固定住后它的完整度消耗会下降。\n \n§7进度：§e%s§7/§e%s\n \n§7基础要求：§e%s\n§7耐久惩罚：§e%s",
```